libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/extensions/NavigationViewContextEx.kt

```diff
@@ -6,14 +6,14 @@ import android.view.ViewGroup
 import androidx.appcompat.widget.AppCompatTextView
```

Comment:
You forgot to update package name in this file.

Reply:
Good catch. Fixed.